lib/elementWrapper/elementWrapper.js

```diff
@@ -49,6 +49,18 @@ class ElementWrapper {
   async elements(retryInterval, retryTimeout) {
     return await _getIfExists(this._get, this._description)(null, retryInterval, retryTimeout);
   }
+
+  async element(index, retryInterval, retryTimeout) {
+    const results = await _getIfExists(this._get, this._description)(
+      null,
+      retryInterval,
+      retryTimeout,
+    );
+    if (index > results.length - 1) {
+      throw new Error(`Element index is out of range. There are only ${results.length} elements`);
```

Comment:
One more small change. I noticed that if there are only 1 element the message can read as follows.

```suggestion
      throw new Error(`Element index is out of range. There are only ${results.length} element(s)`);
```

This way the message will be readable for 1 and more than one elements.